spec/requests/storage_locations_requests_spec.rb

```diff
@@ -65,6 +65,93 @@
 
           expect(response.body.split("\n")[0]).to eq([StorageLocation.csv_export_headers, item3.name, item2.name, item1.name].join(','))
         end
+
+        context "when read_events feature toggle is enabled" do
+          let(:storage_location_with_items) { create(:storage_location, name: "Storage Location with Items", warehouse_type: StorageLocation::WAREHOUSE_TYPES.first) }
+          let(:storage_location_with_duplicate_item) { create(:storage_location, name: "Storage Location with Duplicate Items", warehouse_type: StorageLocation::WAREHOUSE_TYPES.first) }
+          let(:storage_location_with_unique_item) { create(:storage_location, name: "Storage Location with Unique Items", warehouse_type: StorageLocation::WAREHOUSE_TYPES.first) }
+          let(:item1) { create(:item, name: 'A') }
+          let(:item2) { create(:item, name: 'B') }
+          let(:item3) { create(:item, name: 'C') }
+          let(:item4) { create(:item, name: 'D') }
+
+          before do
+            allow(Event).to receive(:read_events?).and_return(true)
+            create(:item, name: 'inactive item', active: false)
+            TestInventory.create_inventory(storage_location_with_items.organization, {
+              storage_location_with_items.id => {
+                item1.id => 1,
+                item2.id => 1,
+                item3.id => 1
+              },
+              storage_location_with_duplicate_item.id => {
+                item3.id => 1
+              },
+              storage_location_with_unique_item.id => {
+                item4.id => 5
+              }
+            })
+          end
+
+          it "generates header with Storage Location fields followed by alphabetized item names" do
+            get storage_locations_path(default_params.merge(format: response_format))
+            expect(response.body.split("\n")[0]).to eq([StorageLocation.csv_export_headers, item1.name, item2.name, item3.name, item4.name].join(','))
```

Comment:
There's no reason to reference `item1.name`etc. when we actually know what the name is (`A`, `B` etc.).

Reply:
Done.